### DISCUSSION TITLE
Configuration support to permit empty

### DISCUSSION THREAD

**Discussion by dbn1234** [spatie/laravel-ciphersweet#42] [Ideas]
Ciphersweet can allow (not throw an exception) when the encrypted field is empty (not part of the query).

This behavior can be controlled via setPermitEmpty(bool) as per : 
https://github.com/paragonie/ciphersweet/issues/83

Note: the cipersweet default is "false".

Can this option be controlled via the laravel-ciphersweet configuration file in a future release?

**Comment by aryala7** [comment]
@dbn1234 this feature was added in the latest release.
